tests/functional/heketi/test_heketi_brick_evict.py

```diff
@@ -111,19 +113,25 @@ def _wait_for_gluster_pod_after_node_reboot(self, node_hostname):
                 self.ocp_client, gluster_pod, service, "active", state)
 
     @pytest.mark.tier4
-    def test_brick_evict_with_node_down(self):
+    @ddt.data("three_nodes", "more_than_three_nodes")
```

Comment:
Use values `(3, '')` as below -
3 - for nodes 3
'' - for more than 3 nodes

Reply:
Done